commands/chargen.py

```diff
@@ -66,12 +66,12 @@ class CmdIC(default_cmds.CmdIC):
     Go in-character (IC) as a given Character.
 
     This will attempt to "become" a different object assuming you have
-    the right to do so. Note that it's the PLAYER character that puppets
+    the right to do so. Note that it's the ACCOUNT character that puppets
```

Comment:
"ACCOUNT character" seems confusing, and is perhaps a hold-over from the original docstring. 

It could probably just be "Note that it's the ACCOUNT that puppets..."

Reply:
Yes, I think this is an artifact of using the rename utility, ha. I tried to catch everything but this slipped through!